Common/Core/RecoDecay.h

```diff
@@ -40,6 +40,22 @@ using namespace o2::constants::math;
 /// - calculation of topological properties of secondary vertices
 /// - Monte Carlo matching of decays at track and particle level
 
+// mapping of charm-hadron origin type
+namespace o2
+{
+namespace aod
+{
+namespace hf_cand
+{
+enum OriginType { None = 0,
+                  Prompt,
+                  NonPrompt };
+} // namespace hf_cand
+} // namespace aod
+} // namespace o2
+
+using namespace o2::aod::hf_cand;
```

Comment:
This should not be needed here as it would be redundant in the files where it is needed and it will pollute the namespace in all other HF files.
How about moving the `OriginType` inside the `RecoDecay` class?

Reply:
Thanks. The additional includes of `RecoDecay.h` in `PWGHF/Tasks` are redundant though, as it is already included in `HFSecondaryVertex.h`.

Reply:
Removed 👍